session/timerq.go

```diff
@@ -56,9 +56,9 @@ func NewTimerQ(nextQueue nqueue) *TimerQ {
 }
 
 // Push adds a message to the TimerQ
-func (a *TimerQ) Push(m *Message) {
+func (a *TimerQ) Push(priority uint64, m interface{}) {
```

Comment:
Why not use an interface here that requires a Priority() method?

Reply:
Firstly, that commit is outdated. Secondly, that question makes no sense and your nqueue interface is wrong because it's Push method needs to have a priority. This cannot and should not be derived from the contents of the message itself. Also... next time look at the latest commit.srsly